src/code/reducers/challengeProgress.js

```diff
@@ -36,17 +36,17 @@ export function getChallengeName(case_, challenge, trial){
   let challengeName = `${case_}:${challenge}:${trial}`;
   return challengeName;
 }
-export function getChallengeScore(case_, challenge, trialCount, progress) {
-  let challengeAwardProgress = [];
-  for (let i = 0; i < trialCount; i++){
-    let challengeProgressName = getChallengeName(case_, challenge, i);
-    let score = progress[challengeProgressName];
-    if (score != null) {
-      challengeAwardProgress.push(score);
-    } else {
-      challengeAwardProgress.push(-1);
-    }
-  }
+// export function getChallengeScore(case_, challenge, trialCount, progress) {
+//   let challengeAwardProgress = [];
+//   for (let i = 0; i < trialCount; i++){
+//     let challengeProgressName = getChallengeName(case_, challenge, i);
+//     let score = progress[challengeProgressName];
+//     if (score != null) {
+//       challengeAwardProgress.push(score);
+//     } else {
+//       challengeAwardProgress.push(-1);
+//     }
+//   }
   
-  return challengeAwardProgress;
-}
\ No newline at end of file
+//   return challengeAwardProgress;
+// }
```

Comment:
Do you want to just delete this all out? If there's stuff that needs to be resurrected, it can always be found in the git history.